src/entry.js

```diff
@@ -15,6 +15,8 @@ if (typeof Components !== 'undefined') {
   // See: https://bugzilla.mozilla.org/show_bug.cgi?id=1007318
   Blob = hiddenWindow.Blob;
   WebSocket = hiddenWindow.WebSocket;
+  FileReader = hiddenWindow.FileReader;
```

Comment:
Ideally, an rtc test that sends binary data, causing the conversion to
happen :)

On Tue Jan 20 2015 at 12:59:52 PM trevj <notifications@github.com> wrote:

> In src/entry.js
> <https://github.com/freedomjs/freedom-for-firefox/pull/45#discussion-diff-23255284>
> :
>
> > @@ -15,6 +15,8 @@ if (typeof Components !== 'undefined') {
> >    // See: https://bugzilla.mozilla.org/show_bug.cgi?id=1007318
> >    Blob = hiddenWindow.Blob;
> >    WebSocket = hiddenWindow.WebSocket;
> > +  FileReader = hiddenWindow.FileReader;
>
> I can add that. Should I also be adding a test in the freedom repo which
> will fail without this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/freedomjs/freedom-for-firefox/pull/45/files#r23255284>
> .
>